internal/handler/impl/host_handler.go

```diff
@@ -13,11 +13,31 @@ import (
 	"gorm.io/gorm"
 )
 
+const (
+	pendoHostConfSuccess = "HostConf"
+	pendoHostConfFailed  = "HostConfFailed"
+)
+
 func (a *application) HostConf(
 	ctx echo.Context,
 	inventoryId public.HostId,
 	fqdn string,
 	params public.HostConfParams,
+) error {
+	err := a.hostConf(ctx, inventoryId, fqdn, params)
+	if err != nil {
+		sendPendoTrackEvent(ctx, a.pendo, pendoHostConfFailed)
+		return err
+	}
```

Comment:
nit: suggestion to express in the form below:

```suggestion
	if err := a.hostConf(ctx, inventoryId, fqdn, params); err != nil {
		sendPendoTrackEvent(ctx, a.pendo, pendoHostConfFailed)
		return err
	}
```

Reply:
Fixed.